build/babel/preset.js

```diff
@@ -44,6 +44,7 @@ module.exports = (context, opts = {}) => ({
     }]
   ],
   plugins: [
+    require('babel-preset-macros'),
```

Comment:
Small typo here: It's not a preset, but plugin ;)

Reply:
NOTE: I'm not stalking you, I was just curious this morning, why macros aren't included in Next.js 7 😄

Reply:
lol Whoops!